### ISSUE TITLE
Can No Longer "SHARE"!!  Please Help--VERY Much Needed

### ISSUE THREAD

**Issue by Chuck-7** [ZachSaucier/Just-Read#254]
So, I have recently been happily sharing away.  This "SHARE" Feature even motivated me to write the Most GLOWING review of Just-Read-Premium, since my work OFTEN requires sharing my research with friends & colleagues.

{{ Please Note!::  The webpages I am sharing via Just-Read Premium, often have just a small segment of the page that I need to Share.  For the remainder of the page, I delete those blocks using the Delete-Block feature in Just-Read. }}

Then TODAY, SHARING simply FAILED!!  Just-Read Premium is allowing me to Share the Page-->
ONLY **_IF_** I did ZERO Edits --- (But then I would need to Share TONS of irrelevant JUNK)!!  After I have done Edits::  I.E.:: Deleting Junk-Blocks, OR Adding a Comment, OR Highlighting Text -- THEN:: __BOOM!!__-->Clicking the SHARE ICON has ZERO EFFECT!!

Please HELP ! ! !!
Thank you Very Much For Your Assistance !!

Chuck Colsch

**Comment by ZachSaucier** [comment]
Hey, sorry to hear that, Chuck. 

Unfortunately I cannot reproduce the behavior that you mention of the share functionality not working with edits being made. Can you try sharing a page like [this one](https://text.npr.org/s.php?sId=718373524) and tell me if it still doesn't work for you?

If it shares okay, what URL did you try sharing on that it failed? Did you try restarting Just Read (I know this will destroy your previous edits -- sorry about that)? 

___
P.S. [I posted](https://github.com/ZachSaucier/Just-Read/issues/4#issuecomment-487644106) your theme on the themes page. Thanks for sharing it!

**Comment by Chuck-7** [comment]
Zach::
Thank you very much for your response!!

Yes — Turns out it had something to do with that SPECIFIC Webpage that I was attempting to Share.  I managed to Share the Page by using Just-Read’s Selection Mode, so it appears that that specific webpage included some weird coding that interacted with Just-Read—which coding Selection-Mode omitted.

**ALSO** thank you for sharing my "Dark Mode With Purple".  I thought my effort at attempting that post made my excellent purple etc. look MUCH crappier than it actually is! Prolly 'cause I was doing the screen-shot improperly.

Allow me to also add that it seems to me that you have under-emphasized the importance of the PREMIUM version of Just-Read (when you reference it on the Non-Premium version's page).  I had used the Non-Premium Version for a LONG time before I carefully Re-read the features of Premium, and thought I should give it a try – – and NOW I consider "Premium" to be extremely important.

So I'll go ahead & "Close" this issue now;  Thank you again for your assistance!
